getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -83,25 +83,28 @@ here's a simple example of how GDScript looks.
         while param2 != 0:
             param2 -= 1
 
-        var local_var2 = param1 + 3
-        return local_var2
-
+        match param3:
+            3: print("param3 is 3!")
+            _:
+                print("param3 is not 3!")
 
-    # Functions override functions with the same name on the base/parent class.
-    # If you still want to call them, use '.' (like 'super' in other languages).
+        var local_var = param1 + 3
+        return local_var
 
+    # Functions override functions with the same name on the base/super class.
```

Comment:
Does this apply to built-in methods such as `_process()`? In 3.2, these are called automatically.

Reply:
Actually, I still need to disable the multilevel call in the code, but that is in the plan. I didn't do it on the main PR because it touches a bunch of code outside GDScript so it should be reviewed independently.